src/Trip.js

```diff
@@ -9,11 +9,11 @@ class Trip {
     this.status = tripsData.status;
     this.suggestedActivities = [];
   }
-// calculate trip cost
   getCostPerTrip(destinationsData) {
     let tripDestination = destinationsData.find(destination => {
       return this.destinationID === destination.id;
     });
+    // console.log(tripDestination);
```

Comment:
this method looks very solid, but something you want to keep in mind is that you want to add 10% on top of the total cost. this will be the `agent's fee`